### PR TITLE
Add more themes from daisyui

### PR DESCRIPTION
**Pull request by StanleyWKMan**
- add more themes
- make theme-selection scrollable
Screenshot 2022-04-25 at 4 59 15 PM

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ export default /** @type {import('tailwindcss/tailwind-config').TailwindConfig}
   },
   plugins: [typography, daisyui],
   daisyui: {
-    themes: ['light', 'dark', 'cupcake', 'emerald', 'valentine', 'synthwave', 'halloween', 'aqua', 'dracula', 'winter']
+    themes: ["light", "dark", "cupcake", "bumblebee", "emerald", "corporate", "synthwave", "retro", "cyberpunk", "valentine", "halloween", "garden", "forest", "aqua", "lofi", "pastel", "fantasy", "wireframe", "black", "luxury", "dracula", "cmyk", "autumn", "business", "acid", "lemonade", "night", "coffee", "winter"],
```
Comment (kwaa) [inline tailwind.config.js:11]: One thing to note is that adding too many themes can look complicated. Later I may re-pick some themes that fit Urara and provide a shortcut for those who want to use all of them.

File: src/lib/components/header.svelte [hunk 2]
```diff
@@ -76,7 +76,7 @@
           </div>
           <ul
             tabindex="0"
-            class="flex shadow-2xl menu dropdown-content bg-base-100 text-base-content rounded-box w-52 p-2 gap-2"
+            class="flex shadow-2xl menu dropdown-content bg-base-100 text-base-content rounded-box w-52 p-2 gap-2 overflow-y-auto h-96"
```
[the remark this answers is not in the document]
Comment (StanleyWKMan) [reply src/lib/components/header.svelte:79]: Thanks for the review! Should the height be longer or shorter? 
Btw gitmoji seems great, will definitely try to use it :)

File: tailwind.config.js (hunk 1 above)
Comment (StanleyWKMan) [reply tailwind.config.js:11]: Sure!

File: src/lib/components/header.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (kwaa) [reply src/lib/components/header.svelte:79]: > Thanks for the review! Should the height be longer or shorter? Btw gitmoji seems great, will definitely try to use it :)

I think a length of about six buttons (including the border) is more appropriate.

[the remark this answers is not in the document]
Comment (StanleyWKMan) [reply src/lib/components/header.svelte:79]: I had to extend the height in tailwind config. I wonder if there is a better way for extending stuff. 
Screenshot 2022-04-27 at 9 05 29 AM

Screenshot 2022-04-27 at 9 07 23 AM

[the remark this answers is not in the document]
Comment (kwaa) [reply src/lib/components/header.svelte:79]: > I had to extend the height in tailwind config. I wonder if there is a better way for extending stuff. Screenshot 2022-04-27 at 9 05 29 AM
> 
> Screenshot 2022-04-27 at 9 07 23 AM

I think you can use square brackets? `h-[21.5rem]`

[the remark this answers is not in the document]
Comment (StanleyWKMan) [reply src/lib/components/header.svelte]: done~

**Pull request merged**